include/groonga/cache.h

```diff
@@ -95,6 +95,21 @@ grn_cache_close(grn_ctx *ctx, grn_cache *cache);
 
 GRN_API grn_rc
 grn_cache_current_set(grn_ctx *ctx, grn_cache *cache);
+/**
+ * \brief Retrieve the current \ref grn_cache object used in the select command.
+ *
+ * \note For more details about the select command, please see
+ * \htmlonly
+ *   <a href="https://groonga.org/docs/reference/commands/select.html">
```

Comment:
Could we use `\sa`?
https://www.doxygen.nl/manual/commands.html#cmdsa

Reply:
fix: [ae2edcd](https://github.com/groonga/groonga/pull/2244/commits/ae2edcd8405c5704df00dc0117303c0b3c3fba2a) Thank you so much. That's the thing I looked for. I used `\see` for consistency in another documentation at this time.